src/zdaemon/tests/tests.py

```diff
@@ -79,7 +79,7 @@ def dont_hang_when_program_doesnt_start():
     >>> write('conf',
     ... '''
     ... <runner>
-    ...   program sleep
+    ...   program sleep .1
```

Comment:
This seems wrong.  AFAIU this example was trying to test what happens if you get an error message from the program (because `sleep` requires an argument and one wasn't provided).  Now you're providing an argument, so `sleep` should no longer fail with an error.